src/node/server.js

```diff
@@ -113,7 +113,7 @@ exports.start = async () => {
     stats.gauge('memoryUsageHeap', () => process.memoryUsage().heapUsed);
 
     process.on('uncaughtException', (err) => {
-      logger.debug(`uncaught exception: ${err.stack || err}`);
+      logger.error(`uncaught exception: ${err.stack || err}`);
```

Comment:
The full stack is already logged with `logger.error` at line 222.

Reply:
The purpose of this PR is to identify the stack trace that's causing the crash. Instances normally don't run on DEBUG logging and in case an error occurs that is not handled and that crashes the instance, we can only hope that no other error was thrown. If there are multiple errors, we don't know which one of the exceptions was fatal.

Reply:
Oh I see what you mean now. Makes sense. I'll close this, maybe I'll add a small note to line 222 "This error was causing the crash"